### PR TITLE
Update XComGame.int Skirmisher Abilities  and Templar Pillar Fixes #549 and Fixes #517 Close #517

### LINKED ISSUE #517: Battlemaster skill description
I'm not sure what this is intended to say. At any rate, it is talking about skirmishers in general, not the skill.

### LINKED ISSUE #549: Small skill description issues
Templar: Changes to Pillar regarding usage is not reflected on it's info screen when promoting.

Skirmisher: Justice claims to end your turn (on the tool-tip for the button to trigger the ability. It does not end your turn, just uses 1 action, even against a non flanked enemy. Pulling an enemy too you, then shooting him, then shooting him again thanks to hit and run is thus possible. Not complaining, exactly, just saying the info should be there to let people know that.

Skirmisher: Body Shield promotion screen information does not give any information about usage: Type of action, limited uses, cool down, etc. Would be helpful to know when deciding to take it or not.

Specialist: Field Surgeon claims that downed and stabilized enemies do not gain a bonus from it. My downed Templar had 2 health in the post action screen, so I am guessing that surgeon is triggering when it shouldn't.

### PR DESCRIPTION
**Pull request by nightmaRok**
Update XComGame.int Skirmisher Abilities #549 and #517

### CODE REVIEW

**Comment by nightmaRok** [comment]
Update XComGame.int Skirmisher Abilities #549 and #517

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 1]
```diff
@@ -4767,16 +4766,29 @@ LocHelpText="Free <Ability:SecondaryWeaponName/> attacks on any enemies that ent
...
+LocHelpText="Use the Grapple to pull a humanoid target to you and deliver a <Ability:SecondaryWeaponName/> strike."
+LocPromotionPopupText="<Bullet/> Justice is a great way to pull an enemy out of position.<br/><Bullet/> The <Ability:SecondaryWeaponName/> strike is guaranteed to hit if the pull is successful.<br/><Bullet/> An open space adjacent to the Skirmisher is needed to use Justice.<br/><Bullet/> Justice has a 4 turn cooldown.<br/><Bullet/> Cost one action point."
 
+[SkirmisherVengeance X2AbilityTemplate]
+LocFriendlyName="Wrath"
+LocLongDescription="Use the Grapple to pull yourself to an enemy and deliver a <Ability:SecondaryWeaponName/> strike."
+LocHelpText="Use the Grapple to pull yourself to an enemy and deliver a <Ability:SecondaryWeaponName/> strike."
+LocPromotionPopupText="<Bullet/> Wrath adds extra mobility for the Skirmisher, and can be used while the Grapple is on cooldown.<br/><Bullet/> The <Ability:SecondaryWeaponName/> strike is guaranteed to hit if the Grapple is successful.<br/><Bullet/> An open space adjacent to the enemy is needed to use Wrath.<br/><Bullet/> Wrath has a 4 turn cooldown.<br/><Bullet/> Cost one action point."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int]: Grapple, Justice and Wrath already have entries in the WOTC localisation files. Why are we adding ones here?

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 2]
```diff
@@ -4767,16 +4766,29 @@ LocHelpText="Free <Ability:SecondaryWeaponName/> attacks on any enemies that ent
...
 
 [Battlemaster X2AbilityTemplate]
 LocFriendlyName="Battlemaster"
-LocLongDescription="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 more aim from high ground, and have significantly lower infiltration times. "
-LocHelpText="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 more aim from high ground, and have significantly lower infiltration times. "
-LocPromotionPopupText="<Bullet/>Skirmisher is the best soldier in ADVENT and XCOM as a whole, and can handle all the company alone.<br/><Bullet/> "
-
-
-
+LocLongDescription="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim from high ground, and have lower infiltration times."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int]: It actually provides +10 aim *and* defense. But I'm thinking of removing Damn High Ground and Tradecraft from the Skirmisher's base perk. But can keep this until then.

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 3]
```diff
@@ -4767,16 +4766,29 @@ LocHelpText="Free <Ability:SecondaryWeaponName/> attacks on any enemies that ent
...
+LocHelpText="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim from high ground, and have lower infiltration times."
+LocPromotionPopupText="<Bullet/> An additional grenade can be equipped in a unique grenade-only slot.<br/><Bullet/> Gain +10 aim from high ground.<br/><Bullet/> This soldier has reduced infiltration times while on missions."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int]: Again, +10 aim *and* defense.

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 4]
```diff
@@ -4874,4 +4886,10 @@ LocLongDescription="The Templar is surrounded by an aura that immediately exting
 LocHelpText="The Templar is surrounded by an aura that immediately extinguishes or blocks any mental impairments for themselves and any nearby squadmates."
 LocFlyOverText="Templar Aura"
 LocPromotionPopupText="<Bullet/> Templar Aura has a 4 tile radius.<br/><Bullet/> Moving next to afflicted allies will immediately cleanse their mental impairments.<br/>"
+
+[Pillar X2AbilityTemplate]
+LocFriendlyName="Pillar"
+LocLongDescription="Summon a pillar of Psionic energy to act as a high cover point. Costs 1 Focus."
+LocHelpText="Summon high cover at a target location. Costs 1 Focus."
+LocPromotionPopupText="<Bullet/> The pillar will last for a number of turns equal to the Templar's Focus level when summoned.<br/><Bullet/> Pillar does not end the Templar's turn.<br/><Bullet/> Can be used after Rend.<br/><Bullet/> Pillar has 1 turn cooldown.<br/><Bullet/> Free action."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int]: Could you put this into the other pull request for #549? I don't really like having a pull request solve multiple independent issues. Especially as there is already another pull request for #549.

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 5]
```diff
@@ -4767,11 +4767,17 @@ LocFlyOverText="Retribution"
...
 ; LWOTC Needs Translation
+LocFriendlyName="Battlemaster"
 LocLongDescription="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim and defense from high ground, and have lower infiltration times."
 LocHelpText="Skirmishers are elite soldiers, due to their past as ADVENT. They have a grenade slot, gain +10 aim and defense from high ground, and have lower infiltration times."
 LocPromotionPopupText="<Bullet/> An additional grenade can be equipped in a unique grenade-only slot.<br/><Bullet/> Gain +10 aim and defense from high ground.<br/><Bullet/> This soldier has reduced infiltration times while on missions."
+
+[SkirmisherGrapple X2AbilityTemplate]
+LocFriendlyName="Grapple"
+LocHelpText="Deploy a grappling hook to move quickly to an elevated position."
+LocLongDescription="Deploy a grappling hook to move quickly to an elevated position."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int]: I don't think you need the above lines. They're the same as vanilla WOTC, right? You only need the one below.

**Comment by pledbrook** [comment]
I rebased and merged manually. Thanks for the contribution!

**Pull request closed without merging**